frontend/src/components/labwork/step/LabworkStep.tsx

```diff
@@ -295,23 +293,17 @@ const LabworkStep = ({ protocol, step, stepSamples }: LabworkStepPageProps) => {
 	const localClearFilters = useCallback((refresh: boolean = true) => {
 		if (clearFilters)
 			dispatch(clearFilters(step.id, refresh))
-	}, [step, step.id])
+	}, [dispatch, step.id])
 
 	const onTabChange = useCallback((tabKey) => {
 		setSelectedTab(tabKey)
-	}, [step.id])
+	}, [])
 
 	useEffect(() => {
 		if (stepSamples.selectedSamples.items.length === 0) {
-			// ensures there are no left over samples from containers that have no selection
-			// because of the abrupt disable of placement tab
-			const sourceContainers = selectSourceContainers(store.getState()).map((c) => c.name)
-			dispatch(flushPlacementContainers(sourceContainers))
-			dispatch(flushLabworkStepPlacementContainers(sourceContainers))
-
 			onTabChange(GROUPED_SAMPLES_TAB_KEY)
```

Comment:
This will trigger twice onTabChange(GROUPED_SAMPLES_TAB_KEY) on clearSelectedSamples. We should remove one of the instances. The one in clearSelectedSamples is likely the easiest to remove as it will consistantly trigger the one here.